ui/imports/shared/views/PasswordView.qml

```diff
@@ -171,7 +180,8 @@ ColumnLayout {
 
         z: root.zFront
         visible: !root.createNewPsw
-        Layout.fillWidth: true
+        Layout.preferredWidth: 416
```

Comment:
The problem is that we are setting a width for the root ColumnLayout which is used by the `PasswordView` and `StatusButton` child components. Therefore the Password field width is entangled with the root Column width.
And because we want more space for the description text, I believe it's fair to remove the coupling in width between the Password fields and the Column similar to https://github.com/status-im/status-desktop/commit/defce32b37a11233d4772d7ea954d325b8d5c4e8 to have similar look and feel as the PasswordConfirmationView for the password field.

I searched for the usage of PasswrodView. It's used in 

PasswordView is used in :
- CreatePasswordView
- CreatePassword
- CreatePasswordModal

And I have verified that the change is consistent across those use cases.

Any thoughts?

Reply:
Ok! I understand!!

I'd create a `d` constant property called, say, `defaultInputWidth: 416` and use it across the component instead of using specific value in here, line `216`and line `319`.